src/context/quizContext.js

```diff
@@ -8,7 +8,9 @@ const quizContext = createContext()
 
 
  const Quizprovider = ({children}) =>{
-     const [ questionCollection ,  setQuestionCollection ] = useState([])
+     const [ questionCollection ,  setQuestionCollection ] = useState([]);
+     const [score, setScore] = useState(0);
+     const [result , setResult ] = useState([]);
```

Comment:
You can  create a function for getting score on clicking correct option for an question.
Name the Context with PascalCase

Reply:
okay , I will do it for the time being , As I have to submit and I am running out of time